# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: accessible/base/nsAccUtils.h
# Commit: 09de143215a1
# Full Hash: 09de143215a13a22dcb2cd53d4ed1dfb60480c82
# Author: Atila Butkovits <abutkovits@mozilla.com>
# Date: 2025-09-30 02:48 +0300
# Regressor Bug: 1987945
# File Overlap Count: 2
# Overlapping Files: accessible/base/nsCoreUtils.cpp, accessible/tests/browser/relations/browser_anchor_positioning.js
# Description:
#   Revert "Bug 1987945: apply code formatting via Lando" for causing bustages at DocAccessible.cpp.
# ==============================================================================

diff -r 826eb4ac00dd -r 09de143215a1 accessible/base/nsAccUtils.h
--- a/accessible/base/nsAccUtils.h	Tue Sep 30 02:35:22 2025 +0300
+++ b/accessible/base/nsAccUtils.h	Tue Sep 30 02:48:50 2025 +0300
@@ -303,13 +303,6 @@
                                      nsCaseTreatment aCaseSensitive);
 
   static bool IsEditableARIACombobox(const LocalAccessible* aAccessible);
-
-  /**
-   * Return true if the CSS positioned target of an anchor is a valid details
-   * related accessible.
-   */
-  static bool IsValidDetailsTargetForAnchor(const Accessible* aDetails,
-                                            const Accessible* aTarget);
 };
 
 }  // namespace a11y